pkg/controller/dgraph/models/group.go

```diff
@@ -0,0 +1,55 @@
+package models
+
+import (
+	"time"
+
+	groups_v1 "github.com/vmware/purser/pkg/apis/groups/v1"
+	"github.com/vmware/purser/pkg/controller/dgraph"
+)
+
+// Dgraph Model Constants
+const (
+	IsGroupCRD = "isGroupCRD"
```

Comment:
there might be another group in kubernetes. I think we can name this as `isPurserGroup`